Dockerfile.debug

```diff
@@ -0,0 +1,41 @@
+# To use GDB inside container run docker like this:
+# docker build . -t pg-lantern-debug
+# docker run --cap-add=SYS_PTRACE -p 5433:5432 -d --name debug-pg pg-lantern-debug
+# Then exec to it docker exec -ti debug-bg bash
+# sudo su # the password is postgres
+# gdb -p $pid_of_pg_backend
+
+FROM ubuntu:22.04
+ARG VERSION=15.3
+
+RUN apt update && apt install wget git make build-essential libreadline8 libreadline-dev zlib1g zlib1g-dev cmake gdb tmux vim sudo -y
+
+# Build and install postgres
+RUN cd /root && wget https://ftp.postgresql.org/pub/source/v${VERSION}/postgresql-${VERSION}.tar.bz2 && \
+    tar xf postgresql-${VERSION}.tar.bz2 && \
+    rm -rf postgresql-${VERSION}.tar.bz2 && \
+    cd postgresql-${VERSION} && ./configure --enable-debug && \
+    make && make install
```

Comment:
Can we build and push up to this point to dockerHUB?
Would be annoying to build postgres from source.

Alternatively, if there is a docker image with postgres installed from source, we could use that

Reply:
Yes I can push that to dockerhub for 15 version and use that image